kano_profile_gui/badge_screen.py

```diff
@@ -217,6 +217,9 @@ def _create_info_box(self):
         description_label.set_line_wrap(True)
         description_label.set_line_wrap_mode(Pango.WrapMode.WORD)
```

Comment:
I did try that and it seems to have no effect at the moment (understandably). Better safe than sorry,  I'll update.